mmedit/models/video_interpolater/basic_interpolater.py

```diff
@@ -0,0 +1,223 @@
+        """
+        losses = dict()
+        output = self.generator(inputs)
+        loss_pix = self.pixel_loss(output, target)
+        losses['loss_pix'] = loss_pix
+        outputs = dict(
+            losses=losses,
+            num_samples=len(target.data),
+            results=dict(
+                inputs=inputs.cpu(), target=target.cpu(), output=output.cpu()))
+        return outputs
+
+    def evaluate(self, output, target):
+        """Evaluation function.
+
+        Args:
+            output (Tensor): Model output with shape (n, c, h, w).
+            target (Tensor): GT Tensor with shape (n, c, h, w).
+
+        Returns:
+            dict: Evaluation results.
+        """
+        crop_border = self.test_cfg.crop_border
+
+        output = tensor2img(output)
+        target = tensor2img(target)
+
+        eval_result = dict()
+        for metric in self.test_cfg.metrics:
+            eval_result[metric] = self.allowed_metrics[metric](output, target,
+                                                               crop_border)
+        return eval_result
+
+    def forward_test(self,
+                     inputs,
+                     target=None,
+                     meta=None,
+                     save_image=False,
+                     save_path=None,
+                     iteration=None):
+        """Testing forward function.
+
+        Args:
+            inputs (Tensor): Tensor of inputs frames with shape (n, c, h, w).
+            target (Tensor): Tensor of target frame with shape (n, c, h, w).
+                Default: None.
+            save_image (bool): Whether to save image. Default: False.
+            save_path (str): Path to save image. Default: None.
+            iteration (int): Iteration for the saving image name.
+                Default: None.
+
+        Returns:
+            dict: Output results.
+        """
+        output = self.generator(inputs)
+        if self.test_cfg is not None and self.test_cfg.get('metrics', None):
+            assert target is not None, (
+                'evaluation with metrics must have target images.')
+            results = dict(eval_result=self.evaluate(output, target))
+        else:
+            results = dict(inputs=inputs.cpu(), output=output.cpu())
+            if target is not None:
+                results['target'] = target.cpu()
+
+        # save image
+        if save_image:
+            self._save_image(meta, iteration, save_path, output)
+
+        return results
+
+    @staticmethod
+    def _save_image(meta, iteration, save_path, pred):
+        if 'target_path' in meta[0]:
+            pred_path = meta[0]['target_path']
+            folder_name = osp.splitext(osp.basename(pred_path))[0]
+        else:
+            pred_path1 = osp.splitext(osp.basename(
+                meta[0]['inputs_path'][0]))[0]
+            pred_path2 = osp.splitext(osp.basename(
+                meta[0]['inputs_path'][1]))[0]
+            folder_name = f'{pred_path1}_{pred_path2}'
```

Comment:
Here I am a bit confused. What are the formats of `target_path` and `inputs_path`? Could you provide an example?